### PR TITLE
7pm 2 update sortcarets

### PR DESCRIPTION
**Pull request by bowenduanmu**
Before this PR, there was custom code to implement the sort carets that appeared at the top of the Count Messages By User table on the  countmessages page of the app. After this PR, that custom code has been removed, because React Bootstrap Table 2 will put these in automatically as long as the correct CSS files are in place. To check this change, go to the countmessages page in the app, and try to sort the table. You should see new sort carets and Bootstrap Table.

### CODE REVIEW

**Review by ujcookcao — DISMISSED** [review]
The new codes replaced the old sortCaret and passed the test.

File: javascript/src/main/components/ChannelMessageReaction/MessageTableReaction.js [hunk 1]
```diff
@@ -23,6 +23,12 @@ export default ({ messages=[],reaction }) => {
...
 
     return (
-        <BootstrapTable keyField='ts' data={messages} columns={columns}/>
+        
+        <BootstrapTable 
+   bootstrap4={true}
+   keyField='id' 
+   data={messages} 
+   columns={columns} 
+/>
```
Comment (andrewhlu) [inline javascript/src/main/components/ChannelMessageReaction/MessageTableReaction.js:32]:
```suggestion
        <BootstrapTable 
            bootstrap4={true}
            keyField='id' 
            data={messages} 
            columns={columns} 
        />
```

Can we fix the spacing for the `BootstrapTable` here (as well as everywhere else in the PR you made this change)?

**Review by andrewhlu — CHANGES_REQUESTED** [review]
Just one small thing about spacing, but otherwise, this is good to go.

File: javascript/src/main/components/ChannelMessageReaction/MessageTableReaction.js (hunk 1 above)
Comment (bowenduanmu) [reply javascript/src/main/components/ChannelMessageReaction/MessageTableReaction.js:32]: Yes. It will be done in next two days.

**Review by pconrad — DISMISSED** [review]
Please update the description.  It should be a description that describes how the *end user* will perceive the change, or how the change impacts the code base, in terms of what's true before and after.

Right now you have:
> remove references to sortCaret and import the CSS styling
  
That's phrased as an instruction to a programmer as to what they should do.  That's fine for an issue, but it is *not* what we are looking for in a PR description. 

It also needs to situate the change in the code base, and/or the  user experience. Your current description tells the reader absolutely nothing about what page of the webapp they should be looking at to test this change.

Instead try something like this. (You'll need to fill in the blanks).

> Before this PR, there was custom code to implement the sort carets that appeared at the top of the _____ table on the _____ page of the app.   After this PR, that custom code has been removed, because React Bootstrap Table 2 will put these in automatically as long as the correct CSS files are in place.  To check this change, go to the ______ page in the app, and try to sort the table.  You should see ______.

File: javascript/src/main/components/ChannelMessageReaction/MessageTableReaction.js (hunk 1 above)
Comment (bowenduanmu) [reply javascript/src/main/components/ChannelMessageReaction/MessageTableReaction.js]: I have fixed all spacing problems.

**Review by ujcookcao — APPROVED** [review]
The code passed all the test and is approved by TA. It LGTM.

**Review by andrewhlu — APPROVED** [review]

**Comment by andrewhlu** [comment]
> The code passed all the test and is approved by TA. It LGTM.

Not sure which TA gave you this verbal approval, but this doesn't really count since it's not an official review on GitHub. I wouldn't be able to normally merge with this.

I have reviewed your code though and it looks okay. Going to merge now.

**Pull request merged**